crypto/rand/rand_unix.c

```diff
@@ -98,9 +106,9 @@ int RAND_poll_ex(RAND_poll_cb rand_add, void *arg)
         /* Get wall clock time, take 8 bits. */
         clock_gettime(CLOCK_REALTIME, &ts);
         v = (unsigned char)(ts.tv_nsec & 0xFF);
-        rand_add(arg, &v, sizeof v, 1);
+        RAND_POOL_add(pool, arg, &v, sizeof(v) , 8);
```

Comment:
Indeed, even more since a few lines above we already claimed only two bits of entropy per byte.
```C
    bytes_needed = RAND_POOL_bytes_needed(pool, 2 /*entropy_per_byte*/);
```